.github/workflows/python-package.yml

```diff
@@ -12,11 +12,11 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
this do not run python 3.6

Reply:
Oh. I see. Okay, reverted back to 20.04.